### DISCUSSION TITLE
GitHub Student Developer Pack :octocat:

### DISCUSSION THREAD

**Discussion by IgnacioPalma** [PadawansBeta/HackademyDia3#5] [General]
:books: Puedes conocer y reclamar tus beneficios acá: [[link]](https://education.github.com/benefits?utm_source=2024-07-08-GITANDGITHUBMENTORINGFORHACKADEMY)

:speech_balloon: ¡Recuerda dejar un comentario en esta discusión para participar por un sorteo de una camiseta de GitHub!

![Copilot Information](https://github.com/BGMP/HackademyDia3/assets/81819758/dc974607-3647-47d1-ad77-ff1767f08aae)

**Comment by llyaregs** [comment]
Estoy muy emocionada por formar parte de git 😼

**Comment by hulkco** [comment]
Muchas gracias!!!!

**Comment by wendyph1209** [comment]
Excelente explicación.. muchas gracias :)

**Comment by Jassit0** [comment]
Me encantó la sesión, la información fue super fluida, y muy clara. Desconocía la extención de copilot que muy seguramente nos van a salvar la vida.  ¡Gracias!

**Comment by V-ac** [comment]
¡Gracias por su tiempo! Explican muy bien
